hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -62,6 +63,8 @@ protected Object run() throws Exception {
 
     if (HystrixCommand.class.isAssignableFrom(method.getReturnType())) {
       return hystrixCommand;
+    } else if (Observable.class.isAssignableFrom(method.getReturnType())) {
+      return hystrixCommand.toObservable();
```

Comment:
Something to consider is portability with a RxJava implementation based on something like RxNetty.  As best I can tell, the Observables in RxNetty are "cold".  

```The request is executed every time {@link HttpClientRequest}, or {@link Observable} returned by {@code write*Content} is subscribed and is the only way of executing the request.```

My knowledge of Hystrix is limited.  Does the cold Observable (`toObervable()`) execute the command each time you subscribe, or does it use a replay, similar to `observe()`?